docs/workflows/transitions/preprocess/preprocess/make_predictions.py

```diff
@@ -59,12 +60,21 @@ def make_predictions(las_client, event):
     model = las_client.get_model(model_id)
     model_metadata = model.get('metadata', {})
     preprocess_config = model.get('preprocessConfig', {})
+    model_field_config = model.get('fieldConfig', {})
     logging.info(f'model metadata: {model_metadata}')
 
     output = {}
     needs_validation = True
 
     labels = get_labels(form_config['config']['fields'])
+    model_labels = get_labels(model_field_config)
+
+    if labels != model_labels:
+        # model has been updated, but form config has not been updated
+        form_config = create_form_config_from_model(model_field_config)
+        labels = model_labels
+        logging.info(f'\nlabels in fieldConfig does not match form_config. Updated form_config is: {form_config}')
```

Comment:
Looks like labels should always be model_labels so there is no reason to conditionally set it inside the if statement.

Also, agree with @magnurud 's comment that we should use what we have in the formConfig for fields that are present in the formConfig and only add the missing fields. We don't want to override the configuration for the other fields in the formConfig such as confidence levels and display name etc.